src/hooks/core/useCoreChatSystemHook.ts

```diff
@@ -15,7 +15,8 @@ import useFetchChannelHook from './chat/useFetchChannelHook';
 import useFetchPostNotificationHook from './chat/useFetchPostNotificationHook';
 import useLocalDatabaseHook from '../../database/hooks/useLocalDatabaseHook';
 import usePostNotificationListenerHook from './getstream/usePostNotificationListenerHook';
-import useProfileHook from './profile/useProfileHook';
+import useSystemMessage from './chat/useSystemMessage';
+import useUserAuthHook from './auth/useUserAuthHook';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `useSystemMessage` and `useUserAuthHook` is a good change as it seems to be replacing the deprecated `useProfileHook`. This should improve maintainability and reduce redundancy in the codebase.
```diff
- import useProfileHook from './profile/useProfileHook';
+ import useSystemMessage from './chat/useSystemMessage';
+ import useUserAuthHook from './auth/useUserAuthHook';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->